app/static/src/app/components/run/RunModelPlot.vue

```diff
@@ -79,13 +81,14 @@ export default defineComponent({
                 };
                 newPlot(el as HTMLElement, baseData.value as Data[], layout, config);
                 (el as EventEmitter).on("plotly_relayout", relayout);
```

Comment:
No, that's not quite what happening here. We don't want to emit a custom event from this component, but to handle the custom plotly event `plotly_relayout` . If I recall, you can't put a standard vue event handler attribute in the template because the element won't recognise the event until after it's been bound to plotly in the previous line. `on` works here, but you need to cast the ement to EventEmitter to keep Typescript happy.